src/components/users/userProfile/EditUserProfile.js

```diff
@@ -0,0 +1,110 @@
+import TextField from "@mui/material/TextField";
+import Button from "@mui/material/Button";
+import React from "react";
+import { useState, useEffect } from "react";
+import client from "../../../utils/client";
+import Header from "../../Header/Header";
+import { useParams } from "react-router-dom";
+import "./style.css";
+
+const UserForm = ({ role }) => {
+  const initialFormData = {
+    firstName: "",
+    lastName: "",
+    email: "",
+    password: "",
+    biography: "",
+    githubUrl: "",
+  };
+
+  const [profile, setProfile] = useState(initialFormData);
+  const [editUserResponse, setEditUserResponse] = useState("");
+
+  const { id } = useParams();
+
+  useEffect(() => {
+    client
+      .get(`/user/${id}`)
+      .then((res) => {
+        setProfile(res.data.data.user);
+      })
+      .catch((err) => console.log(err.response));
+  },);
```

Comment:
this comma isn't needed, in fact I would have thought that this would cause an error 🤔